sklearn/covariance/graph_lasso_.py

```diff
@@ -327,6 +327,33 @@ class GraphicalLasso(EmpiricalCovariance):
     n_iter_ : int
         Number of iterations run.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from scipy import linalg
+    >>> from sklearn.datasets import make_sparse_spd_matrix
+    >>> from sklearn.covariance import GraphicalLasso, log_likelihood
+    >>> n_samples = 100
+    >>> n_features = 10
+    >>> prec = make_sparse_spd_matrix(n_features, alpha=.98,
+    ...                               smallest_coef=.4,
+    ...                               largest_coef=.7,
+    ...                               random_state=0)
+    >>> cov = linalg.inv(prec)
+    >>> d = np.sqrt(np.diag(cov))
```

Comment:
I think this normalisation deserves a comment, and ideally would be g performed with some existing scipy function

Reply:
I found other mistakes in the example. They'd make the example kinda too long, and our docs already show this example. So changed it to be more like the other covariance examples we have, with a manually set sparse covariance matrix, if that's okay.